inabox/tests/payments/reservation_only_test.go

```diff
@@ -0,0 +1,234 @@
+package payments
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	"github.com/Layr-Labs/eigenda/core"
+	"github.com/Layr-Labs/eigenda/core/payments/clientledger"
+	"github.com/Layr-Labs/eigenda/core/payments/reservation"
+	integration "github.com/Layr-Labs/eigenda/inabox/tests"
+	"github.com/Layr-Labs/eigenda/test"
+	"github.com/Layr-Labs/eigenda/test/random"
+	"github.com/stretchr/testify/require"
+)
+
+// NOTE: Currently, it doesn't work to run these tests in sequence. Each test must be run as a separate command.
+// The problem is that the cleanup logic sometimes randomly fails to free docker ports, so subsequent setups fail.
+// Once we figure out why resources aren't being freed, then these tests will be runnable the "normal" way.
+
+func TestReservationOnly_LegacyClient_LegacyController(t *testing.T) {
```

Comment:
Might want to add something like 	t.Skip("Manual test for now"), not sure if the CI is going to try to run the tests automatically